frontend/src/packages/chats/chats-api.ts

```diff
@@ -21,9 +21,9 @@ class ChatApi extends BaseHttpApi {
     super({ path: APIPath.CHATS, baseUrl, http, storage });
   }
 
-  public async getAllChats(): Promise<ChatGetAllResponseDto> {
+  public async getAllChats(query: string): Promise<ChatGetAllResponseDto> {
     const response = await this.load(
-      this.getFullEndpoint(ChatsApiPath.ROOT, {}),
+      this.getFullEndpoint(ChatsApiPath.ROOT, `?query=${query}`, {}),
```

Comment:
I think we have a suitable helper

Reply:
@v0ldemar01 there are nothing for query params. so should i add it?

Reply:
getUrlWithQueryString helper

Reply:
done